test/remote-models.test.js

```diff
@@ -158,4 +178,27 @@ describe('Remote model tests', function() {
         });
     });
   });
+
+  describe('Model find with include filter', function () {
+    it('should return also the included requested  models', function (done) {
```

Comment:
Please move this test to an existing `describe('Model.findById(id, callback)'` block above.

Reply:
Done!